src/main/java/bio/terra/cli/apps/DockerAppsRunner.java

```diff
@@ -107,6 +107,12 @@ public void runToolCommand(
     // check that the current workspace is defined
     workspaceContext.requireCurrentWorkspace();
 
+    // substitute any Terra references in the command
```

Comment:
we should update the javadoc comment on this class, and even consider having a different method for doing terra reference substitution than anything else.
Do we also want to add something to the README on use in a later PR?

Reply:
I added a brief section on the resources commands to the README. But there is already a follow-on ticket [PF-421](https://broadworkbench.atlassian.net/browse/PF-421) for writing a section on the env vars and using references in app commands. I want to use the GLS NF tutorial for that section, but I'm getting a permission error running that. So the follow-on ticket is to debug the permission error and then write the section.